src/sentry/conf/server.py

```diff
@@ -2500,6 +2501,8 @@ def build_cdc_postgres_init_db_volume(settings):
     KAFKA_INGEST_REPLAYS_RECORDINGS: {"cluster": "default"},
     # Metrics Testing Topics
     KAFKA_SNUBA_GENERICS_METRICS_CS: {"cluster": "default"},
+    # User Ip Events -- bucketed by region name
+    KAFKA_REGION_TO_CONTROL: {"cluster": {"_local": "default"}},
```

Comment:
Actually, I found out that there is a way to split consumer from producer configuration here, so this whole thing can be changed and simplified.  coming in next commit